pynestml/codegeneration/resources_nest_desktop/ParameterDeclaration.jinja2

```diff
@@ -0,0 +1,10 @@
+    {%- set name = variable.get_symbol_name() %}
+    {%- set type_symbol = variable.get_type_symbol().print_symbol() %}
+    {%- set default_value = parameters[variable.get_symbol_name()] %}
+      {
+        "id": "{{name}}",
+        "label": "{{name}}",
```

Comment:
Is it possible to take label from the comment part?

Reply:
Done.